app/app.go

```diff
@@ -541,13 +549,30 @@ func NewRollapp(
 		app.AccountKeeper,
 	)
 
-	genesisTransfersBlocker := hubgenkeeper.NewICS4Wrapper(app.ClaimsKeeper, app.HubGenesisKeeper) // ICS4 Wrapper: claims IBC middleware
+	app.HubKeeper = hubkeeper.NewKeeper(
+		appCodec,
+		keys[hubtypes.StoreKey],
+	)
+
+	denomMetadataMiddleware := denommetadata.NewICS4Wrapper(
+		app.ClaimsKeeper,
```

Comment:
@zale144 why we need the claims keeper?
it's unused and planned to be removed asap